packages/plugin-page-view-tracking-browser/test/page-view-tracking.test.ts

```diff
@@ -34,6 +33,7 @@ describe('pageViewTrackingPlugin', () => {
     },
     pageCounter: 0,
   };
+  const defaultPageViewEvent = '[Amplitude] Page Viewed';
```

Comment:
Reuse the existing const?

Reply:
export the defaultPageViewEvent from the plugin-page-view and reuse here.